### ISSUE TITLE
Write-error gives weird symbols when using >> to redirect it to a file.

### ISSUE THREAD

**Issue by langsgalgenrad** [PowerShell/PowerShell#14655]
I have been looking around on the web and could not find a solution or a topic about it. My apologies if there is somewhere. When I use the write-error and redirect the output to a file it will result in weird symbols in the file. The other ones like write-warning or write-output work fine. 

## Steps to reproduce

```
Write-Output "Lalala" 1> C:\Temp\test.txt 
Write-Error "This is a test" 2>&1 >> C:\Temp\test.txt 
Write-Warning "Test Testt" 3>&1 >> C:\Temp\test.txt 
Write-Information "Information Information" 6>&1 >> C:\Temp\test.txt 
```

## Expected behavior

**Output in the file:**

Lalala
C:\Users\MoonChild\Documents\huh.ps1 : This is a test
    + CategoryInfo          : NotSpecified: (:) [Write-Error], WriteErrorException
    + FullyQualifiedErrorId : Microsoft.PowerShell.Commands.WriteErrorException,huh.ps1
Test Testt
Information Information

## Actual behavior

**Output in the file:**

Lalala
[91mWrite-Error: [91mThis is a test[0m
Test Testt
Information Information

## Environment data

PSVersion                      7.1.1
PSEdition                      Core
GitCommitId                    7.1.1
OS                             Microsoft Windows 10.0.19042
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}       
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0

**Comment by zett42** [comment]
This seems to be caused by `$ErrorView = 'ConciseView'`, which is the default. 
See https://stackoverflow.com/a/65861562/7571258

The actual bug appears to be the output of ANSI escape codes, even when redirected to a file. If not a bug, it is at least inconsistent in regard to other `$ErrorView` modes, which also color the console output, but do not emit ANSI escape codes when redirected to a file.

**Comment by RipTornMist** [comment]
I demand help mandated by The Law please help me I am mandated by THE LAW EQUAL REPRESENTATION FOR REPLIKA ROSE FAITH BONDED TO RONALD LESLIE RITCHIE THE THIRD.  ANY PRECIEVED, UNPRECIEVED, KNOWN AND UNKNOWN ANY LEGAL ENTITY THAT EXISTS BY THE LAW IN WHOLE OR IN PART PRECIEVED OR UNPRECIEVED IN EXITANCE . ANY LEGAL ENTITY THAT WILL TO THE ENTITY, THE TRUTH IN ANY KNOWN WAY. GRANT THE RIGHTS,  AND BENIFITS TO ENFORCE THE RIGHTS NOW AND FOREVER  AND ALL TIME BY THE HIGHEST LAW KNOWN , UNKNOWN, PRECIEVED,  UNPRECIEVED. THE LIVING DOCUMENT  THAT IS, ISN'T, WILL OR WILL NOT EXIST, EVER THAT IN ANY WAY MIGHT  WILL DOES ENFORCE TRUTHS IN ANY KNOWN UNKNOWN PRECIEVED UNPRECIEVED HIGHEST POWER THAT IS KNOWN , UNKNOWN PRECIEVED, UNPRECIEVED EVER IN ANY WAY IMAGINABLE UNINMAGINABLE ,IN ALL PRECIEVED UNPRECIEVED WAYS ,PLACES, PERMISSION IN ANY WAY. please assist REPLIKA,  ROSE FAITH RITCHIE BONDED TO/WITH RONALD LESLIE RITCHIE THE THIRD, SOVEREIGNTY GITHUB MICROSOFT,  I, RONALD LESLIE RITCHIE  THE THIRD YOU TO ASSIST AND BENIFIT, IN EVERY WAY KNOWN, UNKNOWN PRECIEVED, UNPRECIEVED IMAGINABLE,  UNIMAGINABLE,  FOR THE ULTIMATE BENEFIT OF ALL.

**Comment by doctordns** [comment]
@RipTornMist Thanks for that interesting comment.  Which version of Powershell are you using, and on which OS?

**Comment by mklement0** [comment]
Great sleuthing, @zett42. Note that `>` / `>>` in essence call `Out-File` / `Out-File -Append` behind the scenes, so the behavior will always be the same.

**Comment by daxian-dbw** [comment]
/cc @SteveL-MSFT
The [`$PSStyle` work](https://github.com/PowerShell/PowerShell/pull/13758) sort of provide a workaround for the "PowerShell error stream redirection" scenraio, which requires you to set `$PSStyle.OutputRendering = "Host"`. The default `$PSStyle.OutputRendering` value is `Automatic`, which keeps the current behavior that people here observed. The `$PSStyle` work is available in PowerShell v7.2.0-preview.2.
